### PR TITLE
Layout set fix

### PR DESCRIPTION
**Pull request by cderb**
Use SetTensorNd from MIOpenDriver.

### CODE REVIEW

File: src/include/conv_fin.hpp [hunk 1]
```diff
@@ -1671,37 +1667,50 @@ int ConvFin<Tgpu, Tref>::ProcessStep(const std::string& step_name)
...
 }
 
+
+template <typename T>
+void PrintVec(std::vector<T> vec)
+{
+    for(auto val: vec)
+        std::cout << val << ' ';
+    std::cout << std::endl;
+}
```
Comment (JehandadKhan) [inline src/include/conv_fin.hpp]:
```suggestion
void PrintVec(const std::vector<T>& vec)
{
    for(const auto& val: vec)
        std::cout << val << ' ';
    std::cout << std::endl;
}
```

File: Jenkinsfile [hunk 2]
```diff
@@ -1,15 +1,15 @@
 
 
 def rocmnode(name) {
-    def node_name = 'rocmtest'
+    def node_name = 'tunatest'
```
Comment (JehandadKhan) [inline Jenkinsfile:4]: Please note, this requires updating the Jenkins config.

File: src/include/conv_fin.hpp [hunk 3]
```diff
@@ -1671,37 +1667,56 @@ int ConvFin<Tgpu, Tref>::ProcessStep(const std::string& step_name)
     return 0;
 }
 
+template <typename T>
+void PrintVec(std::vector<T> vec)
+{
+    for(auto val : vec)
+        std::cout << val << ' ';
+    std::cout << std::endl;
+}
```
Comment (JehandadKhan) [inline src/include/conv_fin.hpp]:
```suggestion
void PrintVec(const std::vector<T>& vec)
{
    for(const auto& val : vec)
        std::cout << val << ' ';
    std::cout << std::endl;
}
```

**Review by JehandadKhan — APPROVED** [review]

**Review by bghimireamd — APPROVED** [review]
LGTM

File: Jenkinsfile (hunk 2 above)
Comment (cderb) [reply Jenkinsfile:4]: This label now matches the Tuna pipe, which places the job on the head node, which has more resources.

**Pull request merged**